docs/2.7.0/docs/daml-finance/tutorials/settlement/transfer.rst

```diff
@@ -0,0 +1,152 @@
+of a holding between Alice and Bob at the Bank. The setup script takes a configuration as input to
+set up Alice's and Bob's account controllers, as outlined in the table above.
+
+The last step of the setup script creates a transfer request of a holding from Alice to Bob:
+
+.. literalinclude:: ../../settlement/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- SETUP_TRANSFER_REQUEST_BEGIN
+  :end-before: -- SETUP_TRANSFER_REQUEST_END
+
+The transfer `Request` template is designed for the stepwise collection of the necessary
+authorizations for transferring a holding to a new owner:
+
+.. literalinclude:: ../../settlement/daml/Workflow/Request.daml
+  :language: daml
+  :start-after: -- TRANSFER_REQUEST_BEGIN
+  :end-before: -- TRANSFER_REQUEST_END
+
+Dual Control
+============
+
+In the `runDualControlTransfer` script, both the custodian and the owner of an account must
+authorize outgoing transfers, while incoming transfers require no authorization.
+
+This script begins by setting up accounts accordingly and creating a transfer request instance:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- SETUP_DUAL_CONTROL_TRANSFER_BEGIN
+  :end-before: -- SETUP_DUAL_CONTROL_TRANSFER_END
+
+To execute the transfer, both the Bank and Alice must authorize:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- DUAL_CONTROL_TRANSFER_BEGIN
+  :end-before: -- DUAL_CONTROL_TRANSFER_END
+
+Discretionary
+=============
+
+The `runDiscretionaryTransfer`` script specifies that the custodian controls both incoming and
+outgoing transfers:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- SETUP_DISCRETIONARY_TRANSFER_BEGIN
+  :end-before: -- SETUP_DISCRETIONARY_TRANSFER_END
+
+Following the setup, the Bank can execute the transfer single-handedly:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- DISCRETIONARY_TRANSFER_BEGIN
+  :end-before: -- DISCRETIONARY_TRANSFER_END
+
+Sovereign
+=========
+
+In the `runSovereignTransfer`` script, the owner controls both incoming and outgoing transfers:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- SETUP_SOVEREIGN_TRANSFER_BEGIN
+  :end-before: -- SETUP_SOVEREIGN_TRANSFER_END
+
+As Alice is the outgoing controller of the sending account, and Bob is the incoming controller of
+the receiving account, both need to authorize the transfer:
+
+.. literalinclude:: ../../quickstart-finance/daml/Scripts/Transfer.daml
+  :language: daml
+  :start-after: -- SOVEREIGN_TRANSFER_BEGIN
+  :end-before: -- SOVEREIGN_TRANSFER_END
+
+Unilateral
+==========
+
+In our final example script, `runUnilateralTransfer`, the owner controls outgoing transfers, while
+incoming transfers require no authorization:
```

Comment:
This might be slightly misleading, as incoming transfers always require the authorization of the `custodian`. In this case, they require no **additional** authorization

Reply:
changed to, "no additional authorization".